modules/gitops-release-notes-1-16-0.adoc

```diff
@@ -0,0 +1,159 @@
+
+* With this update, support is provided for masking sensitive annotations on Secret resources in the Argo CD UI and CLI. A new configuration key, `resource.sensitive.mask.annotations`, has been introduced. This key accepts a comma-separated list of `metadata.annotations` keys. The values associated with these keys are masked in the Argo CD user interface (UI) and command-line interface (CLI), enhancing the security of sensitive information stored in annotations. link:https://issues.redhat.com/browse/GITOPS-5903[GITOPS-5903]
+
+* With this update, support is provided to configure the `respectRBAC` option, which controls how Argo CD watches resources on a cluster in a cluster-scoped installation. Users can now update `respectRBAC` configurations in the `ConfigMap` through an Argo CD resource, allowing for more flexible and granular control over resource viewing behavior. link:https://issues.redhat.com/browse/GITOPS-5212[GITOPS-5212]
+
+* With this update, support is provided for Argo Rollouts monitoring dashboard in the {gitops-title} web console. The Argo Rollouts dashboard provides visibility into key rollout metrics. This dashboard displays information for:
++
+--
+** Total number of active Rollouts resources 
+** Number of available, desired, and unavailable replicas
+** Statistics on the performance of the Rollouts controller 
+--
++
+You can access the Argo Rollouts monitoring dashboard by performing the following steps:
++
+--
+. In the *Administrator* perspective of the web console, go to *Observe* -> *Dashboards*.
+. From the *Dashboard* drop-down list, select the desired {gitops-shortname} dashboard: *{gitops-shortname} /  Rollouts*.
+--
++
+--
+link:https://issues.redhat.com/browse/GITOPS-5161[GITOPS-5161]
+--
+
+* With this update, users can now view the cause of failure directly in the status of Argo CD resources. The error message is clearly provided in the resource status, reducing the need to analyze logs to identify the root cause of failures. link:https://issues.redhat.com/browse/GITOPS-5871[GITOPS-5871]
+
+* With this update, you can access upstream documentation on configuring the ApplicationSet controller in the Argo CD CR to use different policies. For more information, see link:https://argocd-operator.readthedocs.io/en/latest/reference/applicationSet/#applicationset-controller-policies[ApplicationSet Controller policies]. link:https://issues.redhat.com/browse/GITOPS-5236[GITOPS-5236]
```

Comment:
This isn't something we've 'released', AFAICT from looking at the GITOPS entry.

Reply:
Fixed.